.eslintrc.json

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
Can we tag this file and the other eslint as a `mv`? That way we can preserve the git history for this file.